docs/language-server/protocol-language-server.md

```diff
@@ -602,6 +602,15 @@ interface Function {
 
   /** The scope where the function is defined. */
   scope: SuggestionEntryScope;
+
+  /** The documentation string. */
+  documentation?: string;
+
+  /** The rendered HTML of the documentation string. */
+  documentationHtml?: string;
+
+  /** The documentation string divided into sections. */
+  documentationSections?: DocSection[];
```

Comment:
According to [pivotal](https://www.pivotaltracker.com/n/projects/2539304/stories/184117587) the

> LS spend ~5 seconds in `generateDocumentation` method during
> the startup 

What's the impact of expanding of the  suggestion DB by `documentation`, `documentationHtml` & `documentationSections` on the overall size of the transmitted data?

Reply:
`documentationHtml` is a legacy field. I left it for compatibility because I'm not sure if IDE still uses it or not. I planned to remove it when we implement "documentation by request" feature.

> What's the impact ... on the overall size of the transmitted data?

This update adds documentation to local functions and values. We only document the module-level methods and types in the stdlib, so this change should not affect the overall suggestion db size and the startup time